package.json

```diff
@@ -25,32 +25,31 @@
   ],
   "license": "MIT",
   "dependencies": {
+    "bower": "^1.8.2",
```

Comment:
i think this should go in devdeps since this is only to get ember try working, right?

Reply:
Good call. Fixed